tests/TestCase/Http/ResponseTest.php

```diff
@@ -1695,18 +1710,30 @@ public function testWithExpiredCookieOptions()
             'path' => '/custompath/',
             'secure' => true,
             'httpOnly' => true,
-            'expire' => (string)strtotime('+14 days'),
+            'expire' => new \DateTimeImmutable('+14 days'),
```

Comment:
Which DateTime class should this be using? I find the DateTime classes a bit confusing. Should this be `\Cake\I18n\Time`, `Chronos`, `FrozenTime` or `DateTimeImmutable` ?

Reply:
`DateTimeImmutable` is fine here.